app/container.php

```diff
@@ -7,144 +7,6 @@
-};
-
-$c['coverage.dir.phpunit'] = function (Container $c) {
-    return sprintf('%s/%s', $c['coverage.path'], CodeCoverageData::PHP_UNIT_COVERAGE_DIR);
-};
-
-$c['coverage.dir.phpspec'] = function (Container $c) {
-    return sprintf('%s/%s', $c['coverage.path'], CodeCoverageData::PHP_SPEC_COVERAGE_DIR);
-};
-
-$c['phpunit.junit.file.path'] = function (Container $c) {
-    return sprintf('%s/%s', $c['coverage.path'], PhpUnitAdapter::JUNIT_FILE_NAME);
-};
-
-$c['locator'] = function (Container $c): Locator {
-    return new Locator([$c['project.dir']], $c['filesystem']);
-};
-
-$c['path.replacer'] = function (Container $c): PathReplacer {
-    return new PathReplacer($c['filesystem'], $c['phpunit.config.dir']);
-};
-
-$c['test.framework.factory'] = function (Container $c): Factory {
-    return new Factory($c['tmp.dir'], $c['project.dir'], $c['testframework.config.locator'], $c['xml.configuration.helper'], $c['phpunit.junit.file.path'], $c['infection.config'], $c['version.parser']);
-};
-
-$c['xml.configuration.helper'] = function (Container $c): XmlConfigurationHelper {
-    return new XmlConfigurationHelper($c['path.replacer']);
-};
-
-$c['mutant.creator'] = function (Container $c): MutantCreator {
-    return new MutantCreator($c['tmp.dir'], $c['differ'], $c['pretty.printer']);
-};
-
-$c['differ'] = function (): Differ {
-    return new Differ(
-        new BaseDiffer()
-    );
-};
-
-$c['dispatcher'] = function (): EventDispatcher {
-    return new EventDispatcher();
-};
-
-$c['parallel.process.runner'] = function (Container $c): ParallelProcessRunner {
-    return new ParallelProcessRunner($c['dispatcher']);
-};
-
-$c['testframework.config.locator'] = function (Container $c): TestFrameworkConfigLocator {
-    return new TestFrameworkConfigLocator($c['phpunit.config.dir']/*[phpunit.dir, phpspec.dir, ...]*/);
-};
-
-$c['diff.colorizer'] = function (): DiffColorizer {
-    return new DiffColorizer();
-};
-
-$c['test.file.data.provider.phpunit'] = function (Container $c): TestFileDataProvider {
-    return new CachedTestFileDataProvider(
-        new PhpUnitTestFileDataProvider($c['phpunit.junit.file.path'])
-    );
-};
-
-$c['version.parser'] = function (): VersionParser {
-    return new VersionParser();
-};
-
-$c['lexer'] = function (): Lexer {
-    return new Lexer\Emulative([
-        'usedAttributes' => [
-            'comments', 'startLine', 'endLine', 'startTokenPos', 'endTokenPos', 'startFilePos', 'endFilePos',
-        ],
-    ]);
-};
-
-$c['parser'] = function ($c): Parser {
-    return (new ParserFactory())->create(ParserFactory::PREFER_PHP7, $c['lexer']);
-};
-
-$c['pretty.printer'] = function (): Standard {
-    return new Standard();
-};
-
-$c['mutators'] = function (Container $c): array {
-    $mutatorConfig = $c['infection.config']->getMutatorsConfiguration();
-
-    return (new MutatorsGenerator($mutatorConfig))->generate();
-};
-
-return $c;
+return InfectionContainer::getContainer();
```

Comment:
imo in this case this file is not needed and we can move this line right into `bootstrap.php`

Reply:
I'm all in for its removal. Are you sure we can get rid of it?